package.json

```diff
@@ -1,31 +1,34 @@
 {
   "name": "posthtml-content",
-  "description": "transform contents in any way you want",
+  "main": "lib",
   "version": "0.0.0",
-  "author": "static-dev",
+  "engines": {
+    "node": ">=6"
+  },
+  "description": "Content transformations for PostHTML",
+  "scripts": {
+    "prestest": "standard",
+    "test": "nyc ava",
+    "coveralls": "nyc --reporter=lcov ava && cat ./coverage/lcov.info | coveralls"
+  },
   "ava": {
     "verbose": "true"
   },
-  "bugs": "https://github.com/static-dev/posthtml-content/issues",
   "devDependencies": {
     "ava": "0.15.x",
+    "babel-core": "^6.10.x",
+    "babel-preset-es2015": "^6.9.x",
```

Comment:
Same with these ranges, either use `^` or `.x`, not both -- for more, https://nodesource.com/blog/semver-tilde-and-caret/